build.cmd

```diff
@@ -1,7 +1,7 @@
 @echo off
 setlocal
 
-set _args=-restore -build /p:TreatWarningsAsErrors=true %*
+set _args=-restore -build /p:_SelfContainedWasSpecified=true %*
```

Comment:
Is this change still needed? There don't seem to be any warnings to suppress in the latest CI builds.

Reply:
I will revert this and the other change you mentioned after merging. This finally went greenand I want to get this in to unblock other items. Those changes dont' really impact the semantics so should be fine to be in for a few hour.s